http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/GreetingResource.java

```diff
@@ -1,8 +1,18 @@
 package io.quarkus.ts.http.advanced.reactive;
 
+import jakarta.enterprise.inject.spi.CDI;
+import jakarta.ws.rs.Path;
+
 public class GreetingResource extends GreetingAbstractResource {
     @Override
     public String hello() {
         return "Hello from Quarkus REST";
     }
+
+    @Path("/sub-resource")
+    public GreetingSubResources helloFromSubResource() {
+        return CDI.current()
```

Comment:
Why use CDI directly instead of creating a field @Inject annotation? Are there some benefits to doing it here?

Reply:
Ok I see that the issue reproducer use it, so I guess could be related to the issue and you had that motivation to use it.